symplyphysics/definitions/vector/angular_momentum_is_position_cross_linear_momentum.py

```diff
@@ -1,3 +1,16 @@
+r"""
+Angular momentum is position cross linear momentum
+==================================================
+
+The pseudovector of *angular momentum* of a particle is defined relative to a fixed point,
+usually an origin, as a cross product of its position vector and linear momentum.
```

Comment:
Hm, I think vectors are always defined relative to an origin.

Reply:
Not really. The vector of linear momentum, for example, is the same, no matter what origin we choose. I'll reword the description though.

Reply:
I'm guessing you mean the _representation_ of a vector changes depending on the spacial orientation and origin position of the coordinate system, but it's about vector as an entity independent of the coordinate system.